.azure-pipelines/1esmain.yml

```diff
@@ -27,3 +27,5 @@ resources:
 # Use those templates
 extends:
   template: azure-pipelines/1esmain.yml@azExtTemplates
+  parameters:
+    useAzureFederatedCredentials: true
```

Comment:
If you look at the rest of the file, this pipeline file is only run when scheduled for nightly builds. Normal build pipelines that occur on PRs are done through GitHub Actions and not through this file. This is why `useAzureFederatedCredentials` is always set to `true`. For completeness sake, we _could_ also set it to only be true if the build reason is `Schedule`, but that would make it harder to test manually for any reason